backend/icpy/services/filesystem_service.py

```
@@ -1241,6 +1331,102 @@ async def validate_path(self, path: str) -> bool:
+                # Optionally, also emit coalesced individual events on original topic
+                if emit_individual:
+                    for payload in items:
+                        await self.message_broker.publish(topic, payload)
+            except Exception as e:
+                # If batch fails, try to publish individually to avoid data loss
+                logger.debug(f"[FS] Batch publish failed for {topic}: {e}")
+                for payload in items:
+                    try:
+                        await self.message_broker.publish(topic, payload)
+                    except Exception:
+                        pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)